tracer/src/Datadog.Trace/Ci/Sampling/CISampler.cs

```diff
@@ -10,9 +10,9 @@ namespace Datadog.Trace.Ci.Sampling
 {
     internal class CISampler : ISampler
     {
-        public int GetSamplingPriority(Span span)
+        public SamplingDecision MakeSamplingDecision(Span span)
         {
-            return SamplingPriorityValues.UserKeep;
+            return new SamplingDecision(SamplingPriorityValues.UserKeep);
```

Comment:
I assume there's no CI mechanism as it doesn't make sense functionnaly as all CI traces will be kept and as they never interact with prod traces, right?

Reply:
Correct. I inquired about adding a CI mechanism and it was decided that it was not needed. Looks like CI it going to have their own intake anyway separate from tracing.